src/libraries/System.Private.CoreLib/ref/System.Private.CoreLib.csproj

```diff
@@ -0,0 +1,57 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <!-- It is a core assembly because it defines System.Object so we need to pass RuntimeMetadataVersion to the compiler -->
+    <RuntimeMetadataVersion>v4.0.30319</RuntimeMetadataVersion>
+    <!-- disable warnings about obsolete APIs,
+        Remove warning disable when nullable attributes are respected,
+        Type has no accessible constructors which use only CLS-compliant types -->
+    <NoWarn>$(NoWarn);0809;0618;CS8614;CS3015</NoWarn>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <StrongNameKeyId>SilverlightPlatform</StrongNameKeyId>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <FeatureWasmPerfTracing Condition="'$(WasmEnableThreads)' == 'true' or '$(WasmEnablePerfTracing)' == 'true'">true</FeatureWasmPerfTracing>
+    <FeatureWasmThreads Condition="'$(WasmEnableThreads)' == 'true'">true</FeatureWasmThreads>
+    <DefineConstants Condition="'$(FeatureWasmPerfTracing)' == 'true'">$(DefineConstants);FEATURE_WASM_PERFTRACING</DefineConstants>
+    <DefineConstants Condition="'$(FeatureWasmThreads)' == 'true'">$(DefineConstants);FEATURE_WASM_THREADS</DefineConstants>
+    <DefineConstants>$(DefineConstants);HIDE_URI_TYPES</DefineConstants>
+    <DefineConstants>$(DefineConstants);HIDE_CONTRACT_TYPES</DefineConstants>
+  </PropertyGroup>
+  <PropertyGroup>
+    <!-- disable binplacing since the System.Private.CoreLib reference assembly is internal to our build and shouldn't be exposed in ref packs -->
```

Comment:
It wouldn't be exposed in the targeting pack but in the artifacts/bin/ref folder, so anywhere makes more sense here.

```suggestion
    <!-- Disable binplacing since the System.Private.CoreLib reference assembly is internal to our build and shouldn't be exposed anywhere. -->
```